core/api/src/main/java/com/alipay/sofa/rpc/message/AbstractResponseFuture.java

```diff
@@ -27,7 +27,7 @@
  * @author <a href="mailto:zhanggeng.zg@antfin.com">GengZhang</a>
  * @since 5.4.0
  */
-public abstract class AbstractResponseFuture<V> implements ResponseFuture<V> {
+public abstract class AbstractResponseFuture<V> extends ResponseFuture<V> {
```

Comment:
Hi, @dajitui thank you for submitting, but many parts of the internal version rely on this interface. After modifying it to an abstract class, there will be more incompatibilities. See if the interface design can still be preserved.